scripts/india_rbi/below_poverty_line/BelowPovertyLine_India.tmcf

```diff
@@ -0,0 +1,75 @@
+Node: E:BelowPowertyLine_India->E0
+typeOf: dcs:AdministrativeArea1
```

Comment:
This can be just dcs:Place.  That perhaps allows for India-level stat too?

Reply:
Done.